Fix tests that the new version of PHPStan found.

## Problem/Motivation
The Run PHPStan code quality checks step within our ProboCI implementation has found new errors after updating phpstan version.

### Describe the bug
```
1$ mkdir -p /src
2$ cd /src
3$ cd /src/az-quickstart-scaffolding
4$ /src/az-quickstart-scaffolding/vendor/bin/phpstan analyse --configuration web/profiles/custom/az_quickstart/phpstan.neon web/profiles/custom/az_quickstart
5   0/103 [░░░░░░░░░░░░░░░░░░░░░░░░░░░░]   0%
  60/103 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░]  58%
  83/103 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░]  80%
 103/103 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
6
7 ------ --------------------------------------------------------------------- 
8  Line   modules/custom/az_core/tests/src/Functional/ConfigOverrideTest.php   
9 ------ --------------------------------------------------------------------- 
10  14     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See         
11         https://www.drupal.org/node/3083055, which includes recommendations  
12         on which theme to use.                                               
13 ------ --------------------------------------------------------------------- 
14
15 ------ --------------------------------------------------------------------- 
16  Line   modules/custom/az_demo/tests/src/Functional/AZDemoContentTest.php    
17 ------ --------------------------------------------------------------------- 
18  13     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See         
19         https://www.drupal.org/node/3083055, which includes recommendations  
20         on which theme to use.                                               
21 ------ --------------------------------------------------------------------- 
22
23 ------ ------------------------------------------------------------------------ 
24  Line   modules/custom/az_paragraphs/tests/src/Functional/AZParagraphsTest.php  
25 ------ ------------------------------------------------------------------------ 
26  14     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See            
27         https://www.drupal.org/node/3083055, which includes recommendations     
28         on which theme to use.                                                  
29 ------ ------------------------------------------------------------------------ 
30
31 ------ -------------------------------------------------------------------------------------------- 
32  Line   modules/custom/az_paragraphs/tests/src/FunctionalJavascript/AZParagraphsJavascriptTest.php  
33 ------ -------------------------------------------------------------------------------------------- 
34  14     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See                                
35         https://www.drupal.org/node/3083055, which includes recommendations                         
36         on which theme to use.                                                                      
37 ------ -------------------------------------------------------------------------------------------- 
38
39 [ERROR] Found 4 errors                                                         
40
41$ exit
```

--- a/modules/custom/az_core/tests/src/Functional/ConfigOverrideTest.php
+++ b/modules/custom/az_core/tests/src/Functional/ConfigOverrideTest.php
@@ -26,6 +26,11 @@ class ConfigOverrideTest extends BrowserTestBase {
   protected $strictConfigSchema = FALSE;
 
   /**
+   * @var string
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
    * Modules to enable.
    *
    * @var array

--- a/modules/custom/az_demo/tests/src/Functional/AZDemoContentTest.php
+++ b/modules/custom/az_demo/tests/src/Functional/AZDemoContentTest.php
@@ -25,6 +25,11 @@ class AZDemoContentTest extends BrowserTestBase {
   protected $strictConfigSchema = FALSE;
 
   /**
+   * @var string
+   */
+  protected $defaultTheme = 'az_barrio';
+
+  /**
    * Modules to enable.
    *
    * @var array

--- a/modules/custom/az_paragraphs/tests/src/Functional/AZParagraphsTest.php
+++ b/modules/custom/az_paragraphs/tests/src/Functional/AZParagraphsTest.php
@@ -26,6 +26,11 @@ class AZParagraphsTest extends BrowserTestBase {
   protected $strictConfigSchema = FALSE;
 
   /**
+   * @var string
+   */
+  protected $defaultTheme = 'az_barrio';
+
+  /**
    * Modules to enable.
    *
    * @var array

--- a/modules/custom/az_paragraphs/tests/src/FunctionalJavascript/AZParagraphsJavascriptTest.php
+++ b/modules/custom/az_paragraphs/tests/src/FunctionalJavascript/AZParagraphsJavascriptTest.php
@@ -26,6 +26,11 @@ class AZParagraphsJavascriptTest extends WebDriverTestBase {
   protected $strictConfigSchema = FALSE;
 
   /**
+   * @var string
+   */
+  protected $defaultTheme = 'az_barrio';
+
+  /**
    * Modules to enable.
    *
    * @var array

